docker-machine-nfs.sh

```diff
@@ -146,15 +146,15 @@ configureNFS()
     echoSuccess "\t\t\t\t\t\tOK"
 }
 
-# @info:    Configures the Boot2Docker to mount nfs
-configureBoot2Docker()
+# @info:    Configures the Docker Machine to mount nfs
+configureDockerMachine()
```

Comment:
The name of this project is also to 'wide' because it cannot configure NFS on all docker machine drivers. Only boot2docker through the virtualbox driver. I just didn't want to call it `docker-machine-virtualbox-boot2docker-nfs` :p :p :p

That being all said, I think `configureVirtualboxDriver` is a very good alternative as logging message to the console. A lambda user doesn't really have to know that it's boot2docker behind the screens. Though I would keep the method 'configureBoot2Docker' code wise, because that's what it's really about.

What ya think? :)